pkg/agent/datamodel/const.go

```diff
@@ -130,6 +130,101 @@ const (
 	AKSGPUCudaSHA = "sha-ff213d"
 )
 
+/*
```

Comment:
It would also be good to introduce the helpers for GPU SHA like GETAKSGPUIMAGESHA, https://github.com/Azure/karpenter-provider-azure/blob/main/pkg/utils/gpu.go#L33

Reply:
I guess we already have these in the agent package, though unexported...

Reply:
Right we should probably export them, im pretty sure i stole them from you initially

Reply:
Yep I exported them and put them in the new `pkg/agent/common` package for now, but some helpers like `getAKSGPUImageSHA` will only be used for bootstrapping so I might move them to a separate library for self-contained later on.